packages/flame/lib/src/game/mixins/loadable.dart

```diff
@@ -39,6 +39,7 @@ mixin Loadable {
   /// implementing class, it is cached so that it can be reused when the parent
   /// component/game/widget changes.
   @internal
+  @nonVirtual
```

Comment:
Made this to make it non-overridable even inside flame.

Reply:
so like a `final`! cool, TIL